### PR TITLE
Fix the docstring of demos.py

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Fix the docstring of demos.py.

 

### Details

- None

### CODE REVIEW

File: pyvista/demos/demos.py [hunk 1]
```diff
@@ -212,20 +212,20 @@ def plot_wave(fps=30, frequency=1, wavetime=3, interactive=False, notebook=None)
...
+        The desired total display time in seconds.
 
-    interactive : bool, optional
+    interactive : bool, default: False
         Allows the user to set the camera position before the start of the
-        wave movement.  Default ``False``.
+        wave movement.
 
-    notebook : bool, optional
+    notebook : bool, default: None
```
Comment (MatthewFlamm) [inline pyvista/demos/demos.py:228]: This should stay optional if default is correct here and it is unclear what happens in that case to me.  Based on the docstring alone and the docstring of Plotter I'm unsure whether we can just just default of `False`, which would be more clear.

This comment applies to 3 more changes too.

Comment (banesullivan) [reply pyvista/demos/demos.py:228]:
```suggestion
    notebook : bool, optional
```

File: pyvista/demos/demos.py [hunk 2]
```diff
@@ -345,7 +345,7 @@ def plot_ants_plane(notebook=None):
 
     Parameters
     ----------
-    notebook : bool, optional
+    notebook : bool, default: None
```
Comment (banesullivan) [inline pyvista/demos/demos.py:348]:
```suggestion
    notebook : bool, optional
```

File: pyvista/demos/demos.py [hunk 3]
```diff
@@ -387,7 +387,7 @@ def plot_beam(notebook=None):
...
```
Comment (banesullivan) [inline pyvista/demos/demos.py:390]:
```suggestion
    notebook : bool, optional
```

File: pyvista/demos/demos.py [hunk 4]
```diff
@@ -443,7 +443,7 @@ def plot_datasets(dataset_type=None):
 
     Parameters
     ----------
-    dataset_type : str, optional
+    dataset_type : str, default: None
```
Comment (banesullivan) [inline pyvista/demos/demos.py:446]:
```suggestion
    notebook : bool, optional
```

**Review by banesullivan — DISMISSED** [review]

File: pyvista/demos/demos.py [hunk 5]
```diff
@@ -443,7 +443,7 @@ def plot_datasets(dataset_type=None):
 
     Parameters
     ----------
-    dataset_type : str, optional
+    notebook : bool, optional
```
Comment (banesullivan) [inline pyvista/demos/demos.py:446]:
```suggestion
    dataset_type : str, optional
```

**Review by banesullivan — APPROVED** [review]

**Pull request merged**